dist/goldenlayout.js

```diff
@@ -2709,9 +2709,24 @@ lm.utils.copy( lm.controls.Splitter.prototype, {
 	},
 
 	_createElement: function() {
-		var element = $( '<div class="lm_splitter"><div class="lm_drag_handle"></div></div>' );
-		element.addClass( 'lm_' + ( this._isVertical ? 'vertical' : 'horizontal' ) );
-		element[ this._isVertical ? 'height' : 'width' ]( this._size );
+		var dragHandle = $( '<div class="lm_drag_handle"></div>' );
```

Comment:
A lot of changes here seem to be bringing in changes from the `master` that you haven't merged - I'm not certain but can you merge the latest from master and see if that causes these differences to go away? Thanks!